Palace/Book/Models/TPPBookContentType.m

```diff
@@ -5,7 +5,7 @@ TPPBookContentType TPPBookContentTypeFromMIMEType(NSString *const string)
 {
   if ([[TPPOPDSAcquisitionPath audiobookTypes] containsObject:string]) {
     return TPPBookContentTypeAudiobook;
-  } else if ([string isEqualToString:ContentTypeEpubZip]) {
+  } else if ([string isEqualToString:ContentTypeEpubZip] || [string isEqualToString:ContentTypeOctetStream]) {
```

Comment:
@vladimirfedorov, This fix effectively resolves the issue, however, I am still unsure why we are only seeing certain titles in this content mode and only when accessing them by navigating through the catalog. Any ideas?

Reply:
This issue actually is not related to the method of accessing a title. I tracked the issue to the title "The Mirror of Kong Ho" which is actually triggering the crash (TPPBookButtonsView.updateButtons` line 213) due to it's content-mode being the unsupported type `octet-stream`.